release-notes.txt

```diff
@@ -1,3 +1,232 @@
++ minimal GNRC networking example
+
+Build System
+------------
+* split the Cortex-M0 buildtest group to avoid timeout issues with Travis
+* split the Cortex-M4 buildtest group to avoid timout issues with Travis
+
+Other
+-----
++ vagrant configuration
++ documentation: various high-level descriptions of crucial features
++ IoT-LAB: create and connect to debug server
+* pyterm: fix problems with German umlauts as input
+
+Fixed Issues from the last release
+==================================
+
+#2724: Add support for serial number passing to CMSIS boards, document it
+    Documentation about how to discover and set the serial number of CMSIS-DAP chips is missing
+#3201: Odd length packet snips cause invalid check sum
+    If an odd length packet snip occurs in a packet and is not the last snip
+    (in the order the packet is supposed to be, not in the list's order)
+    in a packet it will generate a wrong check sum.
+
+Known Issues
+============
+
+network related issues
+----------------------
+#3075: nhdp: unnecessary microsecond precision
+    NHDP works with timer values of microsecond precision which is not required. Changing
+    to lower precision would save some memory.
+#3086: Max. packet length for AT86RF2XX
+    The size of the link-layer header is not dynamically calculated, but instead the maximum#
+    size is always assumed.
+#3970: RPL: Advertise DODAG only over the assigned interface
+     gnrc_rpl seems to multicast DIOs over all interfaces, though gnrc_rpl_init expects an
+     interface as parameter and sets the RPL-nodes multicast address only for that interface.
+#4048: potential racey memory leak
+     According to the packet buffer stats, flood-pinging a multicast destination may lead to a
+     memory leak due to a race condition. However, it seems to be a rare case and a
+     completely filled up packet buffer was not observed.
+#4462: IPHC/NHC broken between RIOT nodes
+
+native related issues
+---------------------
+#495: native not float safe
+    When the FPU is used when an asynchronous context switch occurs, either the
+    stack gets corrupted or a floating point exception occurs.
+#534: native debugging on osx fails
+    Using valgrind or gdb with a nativenet target in OSX leads to "the network"
+    being stuck (gdb) or the whole process being stuck (valgrind).
+#3341 and #3824: nativenet crashes when hammered
+    Flood-pinging a native instance from more than one host (either multiple threads on the
+    host system or multiple other native instances), leads to a SEGFAULT.
+#4608: tests/xtimer_usleep_until: unstable behaviour
+    The test starts to output "too large difference" and fails after a random period of time.
+
+other platform related issues
+-----------------------------
+#4560: make: clang is more pedantic than gcc
+    oonf_api is not building for clang on OS X
+#4583: cpp11: clang doesn't allow `mutex_t` to be used with `constexpr`
+    All cpp11-* tests fail for OS X with clang.
```

Comment:
I think these both are clang related, I guess they also hold for any other host operating system.